### ISSUE TITLE
Experimental Design - Feedback button accessibility improvement

### ISSUE THREAD

**Issue by andaleliz** [department-of-veterans-affairs/vets-design-system-documentation#2022]
## What
I'd like to propose an experiment with the positioning of the feedback button on the website.  It lives on the far right side of the page, which could be easily missed by folks who are zoomed in.  I wonder if we might be able to position it on the left side of the page and release it to 50% of users behind a feature flag and see what that does for engagement.

## Purpose
Improving accessibility for folks using magnification

## Usage
This component is used across the website.

## Behavior
The feedback button would be anchored on the bottom left corner of the site container, rather than the bottom right.

## Examples
Here's an example of how a page would look on desktop. I moved the back to top button as well, assuming the same guidance applies for magnifiation users + it just looked weird floating out there on the right by itself.

From a code perspective, this would be a fairly easy lift: removing a CSS property from the back to top button, swapping the DOM order of the `last updated` text and feedback button, and removing a CSS class from the parent container of the `last updated` and feedback button.

![button-left-example](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/55992116/034e6ec7-6679-4e16-85e4-4100ae9d6b60)

## Accessibility
See above

## Guidance
N/A

## Research (optional)
[Here are research findings](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/shared-support/accessibility/research/2022-10-a11y-feedback/research-findings.md#key-insights--3) that support placement on the left side of the page being an accessibility improvement

**Comment by caw310** [comment]
@andaleliz the next DSC meeting will be on Friday, Sept 8. This will be the 2nd item on the agenda. I've added you to the meeting invite.

**Comment by andaleliz** [comment]
@caw310 I'll be on PTO Friday so @jfriedrich-va is going to go in my place.  I've added her to the calendar invite.

**Comment by jrfbz** [comment]
Reporting for duty, thanks!

**Comment by briandeconinck** [comment]
Quick summary from the DSC meeting:
- @humancompanion-usds gave the go-ahead for testing this, but cautioned against testing on 50% (suggested 10% or 20% instead)
- Matt also suggested exploring:
   - Similar treatment for Back to Top (it's in the screenshots on the ticket but not discussed)
   - Whether there are any pages on VA.gov where the feedback button will be in close proximity to other primary action buttons, and if switching it to a secondary button would help with that
   - Reaching out to @UXCamG (taking over from Ian) to see if there's anything documented explaining the original decision to put it on the right

**Comment by andaleliz** [comment]
Awesome, thank you @briandeconinck and @jrfbz for representing this issue in the design system council meeting. I will try to hunt down the team who can implement this :) (I think public websites?)

**Comment by andaleliz** [comment]
Update: the feedback button is owned by @clantosswett's team; I've shared this issue with her and she's going to bring it up with her team this week.  She said they can put this in their backlog and determine when they might have the bandwidth to conduct the test.

@caw310 it was suggested that perhaps the design system team owns the code for the Back to Top button and would be the team to implement that piece of the experiment. Is that the case?  Thanks for any info you can provide about that.

Once we have tickets in backlogs for each, I'll close out this ticket.

**Comment by caw310** [comment]
@andaleliz , yes if you can do the work, a DST engineer would review and approve the PR.

**Comment by humancompanion-usds** [comment]
@clantosswett - Did this end up getting assigned or put in the backlog?

**Comment by andaleliz** [comment]
@clantosswett a lot has changed with your teams since I created this last year! Did you ever get this put into a backlog or anything? I've reached out to the a11y specialists to see if anyone would like to shepherd this ticket in my absence.

**Comment by caw310** [comment]
@clantosswett Following up on this request. Did this get added to your backlog?

**Comment by humancompanion-usds** [comment]
Feedback button was changed to a secondary button. Huzzah! Sending over to the Auth Exp team.
